tag-release.sh

```diff
@@ -35,57 +12,70 @@ function current_branch() {
   git rev-parse --abbrev-ref @
 }
 
-if ! branch_exists "${release_branch}" ; then
-  git checkout master
-  if [ ! "$(current_branch)" = master ] ; then
-    echo "Must be on master branch"
-    exit 7
-  fi
-  # Ensure local master is up-to-date by pulling its latest version from origin
-  # and fast-forwarding local master:
-  git fetch origin master
-  git merge --ff-only origin/master
-  # Create the release branch:
-  git push origin master:"${release_branch}"
+#if [[ ! "$(git remote get-url origin)" =~ ^git@github.com:weaveworks/eksctl(\.git)?$ ]] ; then
+#  echo "Invalid origin: $(git remote get-url origin)"
+#  exit 3
+#fi
+
+v=$(go run pkg/version/generate/release_generate.go print-version)
```

Comment:
`go run pkg/version/generate/release_generate.go` has been referenced a few times in this script. It might be worth considering extracting it out into a function that takes the "command" as an argument.